downstream/modules/platform/proc-update-aap-container.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-update-aap-container"]
+
+= Updating container-based {PlatformNameShort}
+
+Perform a patch update for a container-based installation of {PlatformNameShort}.
+
+.Prerequisites
+
+* Reviewed the release notes for the associated patch release.
+
+* Created a backup of your {PlatformNameShort} deployment. For more information, see xref:proc-backup-aap-container[Backing up controller-based {PlatformNameShort}].
```

Comment:
I'd add "You have:" right below the heading to make the bullet points read like complete sentences

Reply:
Done.